app/src/test/java/org/akvo/flow/util/StringUtilTest.java

```diff
@@ -0,0 +1,117 @@
+package org.akvo.flow.util;
+
+import android.test.suitebuilder.annotation.SmallTest;
+import android.text.TextUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+
+/*
```

Comment:
The convention is usually to have the copyright info first, followed by the package and imports etc..